kajas/src/app/profile/profile.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { SessionStorageService } from 'angular-web-storage';
+import { Router } from '@angular/router';
+import axios from 'axios';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrl: './profile.component.css'
+})
+export class ProfileComponent implements OnInit{
+
+  constructor(private sessionStorage: SessionStorageService, private router: Router) {}
+
+  async ngOnInit(): Promise<void> {
+    const url = "http://localhost:4000";
+    const id = this.sessionStorage.get('id');
+    try {
+      const response = await axios.get(`${url}/api/location/id?${id}`);
+      if (response.status === 200) {
+          alert (response.data.msg);
+          console.log('Navigating to /setupprofile');
```

Comment:
### Remove console.logs() and unneccessary codes in the production environment.
- Use console.error() for logging error messages.
- Remove codes/variables that are not used.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2637aafa621_0_0